src/nl/hannahsten/texifyidea/inspections/latex/LatexCollapseCiteInspection.kt

```diff
@@ -41,8 +47,8 @@ open class LatexCollapseCiteInspection : TexifyInspectionBase() {
                 continue
             }
 
-            val bundle = cmd.findCiteBundle()
-            if (bundle.size <= 1) {
+            val bundle = cmd.findCiteBundle().filter { it.optionalParameterMap.isEmpty() }
+            if (bundle.size <= 1 || !bundle.contains(cmd)) {
```

Comment:
```suggestion
            if (bundle.isEmpty() || !bundle.contains(cmd)) {
```

Reply:
This is not equivalent, it would have been if we checked for *strictly* less than 1.

Reply:
I'll change it into `< 2` for clarity.

Reply:
lol yes